xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -20,8 +20,11 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Reads bytes as primitives with specific endianness</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[For native formats, MemoryExtensions.Read{T}; should be used.
```

Comment:
@layomia @JeremyKuhne @ahsonkhan can you chime in? Were we supposed to have Read methods in MemoryExtensions? Maybe we planned to have them but then we pulled back?

Reply:
@GrabYourPitchforks may also know what happened here, since he has been involved in most of the Buffers.Binary changes. Do you know why this remark was added? Should we remove it since there's no MemoryExtensions.Read method?

Reply:
This API was moved from MemoryExtensions to MemoryMarshal.

So, the docs should read `MemoryMarshal.Read{T}`

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.memorymarshal.read?view=netstandard-2.1

At the time that remark was written, we had the Read method on SpanExtensions (which is MemoryExtensions used to be called) and we never updated it with the move from corefxlab to corefx:
https://github.com/dotnet/corefxlab/blob/21109afa5ffdc3bbab08e36e9c4b01ee5834c5e9/src/System.Slices/System/SpanExtensions.cs#L231

Reply:
We could very well just remove that remark, also because I don't know if "for native formats" makes sense. We could consider adding a "see also `MemoryMarshal.Read{T}`", but not worth it, imo.

Reply:
Okay, I removed it. Thanks.